.github/workflows/update-openapi-ui.yml

```diff
@@ -51,7 +56,7 @@ jobs:
         git commit -m "Update Swagger UI to $SWAGGER_UI_VERSION [auto]"
         git push
     - name: Create Pull Request
-      uses: peter-evans/create-pull-request@18f7dc018cc2cd597073088f7c7591b9d1c02672 #3.14.0
+      uses: peter-evans/create-pull-request@v5.0.2
```

Comment:
It was brought up before that using a version allows the author of the action to make nefarious changes to that version which we would then consume.

Reply:
Have replaced version with the commit sha